src/osd/osd_types.cc

```diff
@@ -370,6 +373,7 @@ void osd_stat_t::generate_test_instances(std::list<osd_stat_t*>& o)
   o.back()->kb = 1;
   o.back()->kb_used = 2;
   o.back()->kb_avail = 3;
+  o.back()->pg_sum = 100;
```

Comment:
You shouldn't add this to the osd_stat_t.  It breaks compatibiltiy (at least the way it was added does) but more importantly it's not needed--the monitor already has a per-osd pg sum in the PGMap (see mon/PGMap.h).  Use that instead!